packages/forms/demo/fileList.stories.mdx

```diff
@@ -21,10 +21,16 @@ import { FILELIST_ITEMS as ITEMS } from './stories/data';
 <Canvas>
   <Story
     name="FileList"
-    args={{ items: ITEMS }}
+    args={{
+      items: ITEMS,
+      closeAriaLabel: 'Close',
+      deleteAriaLabel: 'Delete'
+    }}
     argTypes={{
       items: { name: 'children' },
-      isCompact: { control: 'boolean', table: { category: 'File' } }
+      isCompact: { control: 'boolean', table: { category: 'File' } },
+      closeAriaLabel: { name: 'aria-label', table: { category: 'File.Close' } },
+      deleteAriaLabel: { name: 'aria-label', table: { category: 'File.Delete' } }
```

Comment:
[nit] these wouldn't necessarily _have_ to live here, since we are limiting the surface of the `File` controls to only things that affect the FileList. The strings could just be hard-coded in the `FileListStory`.

Reply:
Fixed: https://github.com/zendeskgarden/react-components/pull/1509/commits/3ef82da89b72839b6415eccfdb9e458bce1e4d9d

Reply:
That's a fair point, I'll remember that for next time re: patterns / generalized examples. :)